.github/actions/install-dpcpp/action.yml

```diff
@@ -36,32 +36,66 @@ runs:
           URL=https://github.com/intel/llvm/releases/download/${{ inputs.DPCPP_VERSION }}/sycl_linux.tar.gz;
         else
           echo "Will use latest DPCPP version"
-          latest=$(curl -sS https://api.github.com/repos/intel/llvm/releases | jq -r '[.[].tag_name|select(match("nightly-[0-9]{4}-[0-9]{2}-[0-9]{2}"))][0]')
+          # Use grep and sed instead of jq to avoid additional dependencies
+          latest=$(curl -sS https://api.github.com/repos/intel/llvm/releases | \
+            grep '"tag_name":' | \
+            grep 'nightly-[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}' | \
+            head -1 | \
+            sed 's/.*"tag_name": *"\([^"]*\)".*/\1/')
           URL=https://github.com/intel/llvm/releases/download/${latest}/sycl_linux.tar.gz;
         fi
         echo "Downloading DPCPP from ${URL}"
         sudo wget -q $URL
         sudo tar -xf sycl_linux.tar.gz
         sudo rm sycl_linux.tar.gz
         popd
+        
+        # Detect system C++ compiler and standard library automatically
```

Comment:
what's the reason to do this? DPC++ automatically detects the system c++. do you want a different version than dpc++ detects or does dpc++ fail to detect?